packages/css/src/types.ts

```diff
@@ -1,15 +1,24 @@
 import type { MapLeafNodes, CSSVarFunction } from '@vanilla-extract/private';
-import type { PropertiesFallback, AtRule, Properties } from 'csstype';
+import type { Properties, AtRule } from 'csstype';
 
 import type { SimplePseudos } from './simplePseudos';
 
-type CSSTypeProperties = PropertiesFallback<number | (string & {})>;
+// csstype is yet to ship container property types as they are not in
+// the output MDN spec files yet. Remove this once that's done.
```

Comment:
It might be worth a link to the MDN docs here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries